### ISSUE TITLE
CSV line recovery forces Windows line endings

### ISSUE THREAD

**Issue by sinus-x** [certtools/intelmq#1597]
The `recover_line_csv()` function is joining data with `\r\n`, redgardless of the source file newlines:
https://github.com/certtools/intelmq/blob/a2d20df6fd4fa0386fe79e66156537336faf92b0/intelmq/lib/bot.py#L1042

This means that bot test are failing even though the parsing succeeded, because the `raw` values do not match: there is a new CR byte. As a workaround, bot tests are replacing newlines in their test files to account for that, which is not ideal solution.
https://github.com/certtools/intelmq/blob/f7a5c7adb9e6a3b8a65c6e453195fd914fe80c20/intelmq/tests/bots/parsers/generic/test_parser_csv_data_type.py#L20

**Comment by wagner-certat** [comment]
Yes, ideally the use line ending is detected in `ParserBot.parse_csv` and `ParserBot.parse_csv_dict` and then saved in the class or somewhere else.
